documentation/versions-and-deploys.md

```diff
@@ -4,39 +4,37 @@ Shopify’s UI extensions are a [versioned API](https://shopify.dev/api/usage/ve
 
 The `changeset` command will then ask you for a short description of your change. Please try to make the description clear and helpful for other developers attempting to understand the changes you have made. Once you commit your description, the command will create a markdown file in the `.changeset` directory at the root of this repo, which contains the details you provided. If your change needs additional detail (for example, you want to show code examples or migration instructions), you can add that content to the markdown file. The contents of these files will be “consumed” when we publish the next version of the package, and will be used to generate a changelog for the packages you changed.
 
 Commit your changeset file alongside the rest of the changes you are making — the code and changeset should be reviewed as part of the same PR.
 
-> **Note**: If your PR changes several things, consider separate changesets for each commit.
-
 ## Deploying new versions to NPM
 
 When changeset files are detected on any of the stable version branches, a GitHub action will create a new PR that merges all the unpublished changesets together into a single changelog, and increments the version number on updated packages. This PR should be reviewed and merged by a member of the [UI Extension Stewards GitHub team](https://github.com/orgs/Shopify/teams/ui-extension-stewards). Once it is merged, the new package versions will automatically be published to NPM, with a tag corresponding to their API version (for example, `yarn add @shopify/ui-extensions@2023-01` will install the latest package version for the `2023-01` API version).
 
 The `unstable` branch does not have the additional step of creating a PR to merge changesets “on-demand”. Instead, _every_ merge into `unstable` causes a new set of versions to be published to NPM. A GitHub action watches this branch and, when any changes are detected, it will create a [“snapshot release”](https://github.com/changesets/changesets/blob/main/docs/snapshot-releases.md) that publishes the changed packages as they exist at that point in time. Snapshot versions have the format `0.0.0-unstable-{{TIMESTAMP}}`, and are published to NPM with the `unstable` tag. This allows developers to get the latest `unstable` version of a package by running an install command referencing the `unstable` tag, like `yarn add @shopify/ui-extensions@unstable`.
 
+Like the `unstable` branch, the `internal` branch also publishes snapshot releases to NPM. These releases are identified by the format `0.0.0-internal-{{TIMESTAMP}}` and are published with the `internal` tag. It's important to note that these releases are intended for use by Shopify developers only and **are not intended for external use**, as breaking changes may be introduced without prior notice.
+
```

Comment:
I think you want to add this part about "internal", but don't remove any of the other changes. This README was updated in `unstable` and looks like this is overwriting it.

Reply:
oops. yep. Fixed it! :)